### DISCUSSION TITLE
Suggestion de fonctionalité supplémentaire

### DISCUSSION THREAD

**Discussion by xbertrand** [marian-m12l/studio#167] [Ideas]
Bonjour Marian,
Encore un grand merci pour ce logiciel, mon fils est ravi de retrouver dessus ses chansons et poésies de l'école !
En créant un pack avec ces poésies et comptines, j'ai utilisé les noeuds simplifiés. Dans un noeud de menu, j'ai voulu inverser l'ordre de certaines histoires. mais il faut en fait tout refaire (image et titre de toutes les histoire). Avec les noeuds expert, c'est plus simple, il suffit de refaire les liens qui sortent du noeud d'action.
Je me disais donc qu'une amélioration pouvait être de pouvoir bouger les "boites" dans le noeud de menu, pour les ordonner plus facilement après les avoir créés dans un mauvais ordre.
N'étant pas calé en programmation, je ne sais pas du tout si c'est compliqué à faire. En tout cas le logiciel est vraiment top comme ça.
Encore merci !!
Bonne journée .
Xavier

**Comment by marian-m12l** [comment]
@xbertrand Merci pour le retour :-) Ce serait une fonctionnalité bien utile ! J'ai dû l'envisager à un moment mais repousser l'implémentation à plus tard... Je me note ça.
